Description: Some folder are not created before installing files into them:
 fix that. Also disables the udev rules reload (useless on buildd).
Author: Didier Raboud <didier@raboud.com>
Origin: vendor
Forwarded: yes (private mail)
Last-Update: 2009-01-28
Index: usb-modeswitch.git/Makefile
===================================================================
--- usb-modeswitch.git.orig/Makefile	2010-01-25 11:00:24.000000000 +0100
+++ usb-modeswitch.git/Makefile	2010-01-25 11:01:31.000000000 +0100
@@ -22,14 +22,16 @@
 clean:
 	$(RM) ./usb_modeswitch
 
-install: all files-install rules-reload
+install: all files-install
 
 files-install:
 	install -d $(SBINDIR)
 	install --mode=755 ./usb_modeswitch $(SBINDIR)
+	install -d $(UDEVDIR)
 	install --mode=755 ./usb_modeswitch.sh $(UDEVDIR)/usb_modeswitch
 	install -d $(ETCDIR)/usb_modeswitch.d
 	install --mode=644 -t $(ETCDIR)/usb_modeswitch.d ./usb_modeswitch.d/*
+	install -d $(MANDIR)
 	install --mode=644 usb_modeswitch.1 $(MANDIR)
 	mkdir -p $(RULESDIR)
 	install --mode=644 40-usb_modeswitch.rules $(RULESDIR)
@@ -50,7 +52,7 @@
 		`which udevcontrol` reload_rules; \
 	fi
 
-uninstall: files-uninstall rules-reload
+uninstall: files-uninstall
 
 files-uninstall:
 	$(RM) $(SBINDIR)/usb_modeswitch
